docs/src/conformal_cubed_sphere.md

```diff
@@ -54,39 +64,41 @@ colgap!(fig.layout, 40)
 current_figure()
 ```
 
-Above, we plotted the mapping from the cube's face onto the sphere both in a 2D
-projection (e.g., overlooking the sphere down to its North Pole) and in 3D space.
+Above, we plotted the mapping from the cube's face onto the sphere both in a 2D projection (e.g., overlooking the sphere down to its North Pole) and in 3D space.
 
-We can then use [Rotations.jl](https://github.com/JuliaGeometry/Rotations.jl) to rotate
-the face of the sphere that includes the North Pole and this way obtain all six faces of
-the sphere.
+We can then use [Rotations.jl](https://github.com/JuliaGeometry/Rotations.jl) to rotate the face of the sphere that includes the North Pole and this way obtain all six faces of the sphere.
 
 ```@example 1
 using Rotations
 
-fig = Figure(resolution = (800, 400))
+colors = [:purple, :red, :orange, :cyan, :green, :blue]
```

Comment:
prescribing the colors is not really needed right?

(I'm just saying this in an attempt to simplify)